bigquery/scheduled_queries/calculate-daily-mat-time-metrics.sql

```diff
@@ -0,0 +1,346 @@
+  #scheduled query that appends new rows to CALCULATED_daily_MAT_time_metrics table
+WITH
+  dates AS ( #the range of dates we're calculating for
+  SELECT
```

Comment:
How strict are .sql files about indentation? My eyes are looking for where this bracket closes. Is it line 270?

Reply:
Maybe it's 14. If possible please indent the contents of the brackets by 1, for readability.

Reply:
Essentially, BigQuery has a SQL autoformatter that (if you use it) controls the indentation. Looking at it it seems to prefer intending based on the logic within the keywords, rather than on brackets, which I suspect is a bit of a SQL thing as it's not a scripting language. Although different people's SQL differs, and personally I'd favour as much indentation as possible to make things clearer.

So essentially... either we have more indentation but lose the auto format function, or we do without the extra indentation but keep the auto formatter (which is a big time saver).